src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -72,5 +72,18 @@
     
     <bean id="arEmailService" parent="arEmailService-parentBean"
         class="edu.cornell.kfs.module.ar.service.impl.CuAREmailServiceImpl" />
+        
+    <bean id="ArAuthorizationConstants" class="org.kuali.kfs.core.api.util.collect.ConstantsMap"
```

Comment:
Should comment(s) be added here to indicate that this bean and the following one were backported?

Reply:
Comment added